lib/qbo_api.rb

```diff
@@ -15,7 +15,7 @@
 
 class QboApi
   extend Configuration
-  include Connection
+  prepend Connection
```

Comment:
I'm on a recent ruby and don't need this.  where you seeing a problem?

Reply:
This is the fix for #87, since `QboApi::Connection::OAuth1#default_attributes` and `QboApi::Connection::OAuth2#default_attributes` aren't ever accessed (included modules don't supersede or overwrite existing methods).

It seems like those are only used for the constants you dislike, so you're welcome to refactor it however you like. We could consider deprecating those constants, but I do like how it enables you to set defaults for the app ID/secret/etc.

Reply:
Let's address that issue separately from this PR.

I understand how modules work, so if it doesn't work right, it's a bug.

But really, instead of isolated behavior in files and modules,  I'd like to have different connection strategies.

It's hinted at in ` Connection.add_authorization_middleware`

Reply:
@bf4 What version of Ruby are you on? And does `prepend Connection` work with it?